freeze.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+set -e
+set -x
+
+source manifest
+
+# Allow multiple downloads
+sed -i '/ParallelDownloads/s/^#//g' /etc/pacman.conf
+# set archive date if specified and force update/downgrade builder
+if [ -n "${ARCHIVE_DATE}" ]; then
+	echo "
+	Server=https://archive.archlinux.org/repos/${ARCHIVE_DATE}/\$repo/os/\$arch
+	" > /etc/pacman.d/mirrorlist
+fi
+
+# Update system and install pikaur and other build dependencies
+echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist\n" >> /etc/pacman.conf && \
+	pacman --noconfirm -Syyu && \
+	pacman --noconfirm -S \
+		arch-install-scripts \
+		btrfs-progs \
+		pyalpm sudo \
+		wget \
+		xcb-util-wm \
+		fmt \
+		python-markdown-it-py \
+		python-wheel \
+		python-build \
+		python-installer \
+		python-setuptools \
+		&& \
+	pacman --noconfirm -S --needed git && \
+	echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+	useradd build -G wheel -m && \
+	su - build -c "git clone https://aur.archlinux.org/pikaur.git /tmp/pikaur" && \
+	su - build -c "cd /tmp/pikaur && makepkg -f" && \
+	pacman --noconfirm -U /tmp/pikaur/pikaur-*.pkg.tar.zst
```

Comment:
Any specific reason to do it in a multiline one-liner? Since the script is run with `set -e` any command that fails will have the same effect.